### PR TITLE
Add a new table to list and describe supported log types for detector creation

### LINKED ISSUE #4945: [DOC]Create new page for supported log types in Security Analytics
**What do you want to do?**
 
- [x] Request a change to existing documentation
- [x] Add new documentation
- [ ] Report a technical problem with the documentation
- [ ] Other 

**Tell us about your request.**
There is no dedicated reference that users can consult to find out what log types are currently supported for Security Analytics. They are listed on the Security Analytics landing page. But this list simply names them. Detector creation steps provide a screen shot that lists them. But it's not an adequate source of information. This creates a new page for supported log types.
 

**What other resources are available?**
"log types and rules" screenshot that appears in [defining a detector](https://opensearch.org/docs/latest/security-analytics/sec-analytics-config/detectors-config/#step-1-define-a-detector).

### PR DESCRIPTION
**Pull request by cwillum**
### Description
Added new topic that lists supported log types.

### Issues Resolved
Fixes #4945 

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by cwillum** [comment]
Contacted team members for tech review and additional input on descriptions of log types.

**Review by amsiglan — APPROVED** [review]

**Comment by cwillum** [comment]
@amsiglan Big thanks.

File: _security-analytics/sec-analytics-config/log-types.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+| **Amazon S3 access logs** | These logs track requests for access to an Amazon S3 bucket.
+| **Google Workspace logs** | Logs for Google Workspace that can monitor log entries such as admin actions, group and group membership actions, and events having to do with log in.   
+| **GitHub actions** | Logs that monitor workflows created by [GitHub Actions](https://docs.github.com/en/actions/learn-github-actions/understanding-github-actions).
+| **Microsoft 365 logs** | Microsoft 365 audit logs collect a range of data for Microsoft 365 including records from call details, performance data, SQL Server, security events, and access control activity.
+| **Okta events** | These logs record Okta events from a range of actions such as downloading an export file, requesting an application access, or revoking privileges. 
+| **Microsoft Azure logs** | Logs that monitor log data for Cloud applications managed by the Microsoft Azure cloud service platform.
```
Comment (hdhalter) [inline _security-analytics/sec-analytics-config/log-types.md:27]:
```suggestion
| **Microsoft Azure logs** | Logs that monitor log data for cloud applications managed by Microsoft Azure Cloud Services.
```

File: _security-analytics/sec-analytics-config/log-types.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+| **VPC Flow logs** | [VPC Flow logs](https://docs.aws.amazon.com/prescriptive-guidance/latest/logging-monitoring-for-application-owners/vpc-flow-logs.html) capture information about the IP traffic going to and from network interfaces in your VPC.
```
Comment (hdhalter) [inline _security-analytics/sec-analytics-config/log-types.md:28]:
```suggestion
| **VPC Flow logs** | [VPC Flow Logs](https://docs.aws.amazon.com/prescriptive-guidance/latest/logging-monitoring-for-application-owners/vpc-flow-logs.html) capture information about the IP traffic going to and from network interfaces in your VPC.
```

**Review by hdhalter — APPROVED** [review]
Just a few nits. Looks good!

**Pull request merged**